app/src/main/java/io/intrepid/contest/models/Participant.java

```diff
@@ -0,0 +1,29 @@
+package io.intrepid.contest.models;
+
+import java.util.Date;
+import java.util.UUID;
+
+public class Participant {
+    private UUID id;
```

Comment:
There are a lot of unused variables here. I will leave it up to you two to decide whether we want to keep them in or remove them until we actually need them.